src/porepy/models/constitutive_laws.py

```diff
@@ -2571,6 +2662,49 @@ def biot_stabilization(self, subdomains: list[pp.Grid]) -> pp.ad.Operator:
         return stabilization
 
 
+class BiotPoromechanicsPorosity(PoroMechanicsPorosity):
```

Comment:
With the refactoring, this is a quite neat implementation of Biot! And thanks for the thorough documentation!

Reply:
Not a problem. Thanks for the idea.